socorro/external/es/super_search_fields.py

```diff
@@ -1689,6 +1699,11 @@ def number_field(
         "name": "em_check_compatibility",
         "namespace": "raw_crash",
         "permissions_needed": [],
+        "source_key": "processed_crash.em_check_compatibility",
+        "destination_keys": [
+            "raw_crash.EMCheckCompatibility",
+            "processed_crash.em_check_compatibility",
```

Comment:
I should add validation of this to the tests. That'll mean I don't have to eyeball this very carefully.